ccm_web/client/src/components/SingleSectionEnrollmentWorkflow.tsx

```diff
@@ -0,0 +1,334 @@
+    )
+  }
+
+  const getSelectContent = (): JSX.Element => {
+    if (props.getSectionsError !== undefined) {
+      return (
+        <ErrorAlert
+          messages={[<Typography key={0}>An error occurred while loading section data from Canvas.</Typography>]}
+          tryAgain={async () => {
+            setSelectedSection(undefined)
+            await props.doGetSections()
+          }}
+        />
+      )
+    }
+
+    return (
+      <>
+      <div className={classes.container}>
+        <CreateSelectSectionWidget
+          sections={props.sections}
+          selectedSection={selectedSection}
+          setSelectedSection={setSelectedSection}
+          // Only admins have access to the Add UM Users feature, and they can create sections.
+          canCreate={true}
+          course={props.course}
+          onSectionCreated={(s) => {
+            setSelectedSection(injectCourseName([s], props.course.name)[0])
+            props.onSectionCreated(s)
+          }}
+        />
+        <Backdrop className={classes.backdrop} open={props.isGetSectionsLoading}>
+          <Grid container>
+            <Grid item xs={12}><CircularProgress color='inherit' /></Grid>
+            <Grid item xs={12}>Loading section data from Canvas</Grid>
+          </Grid>
+        </Backdrop>
+      </div>
+      <Grid container className={classes.buttonGroup} justifyContent='space-between'>
+        <Button variant='outlined' aria-label='Back to select input method' onClick={props.resetFeature}>
+          Back
+        </Button>
+        <Button
+          variant='contained'
+          color='primary'
+          disabled={selectedSection === undefined}
+          onClick={() => setActiveStep(CSVWorkflowStep.Upload)}
+        >
+          Select
+        </Button>
+      </Grid>
+      </>
+    )
+  }
+
+  const renderRowValidationErrors = (errors: RowValidationError[]): JSX.Element => {
+    return (
+      <>
+      {file !== undefined && <CSVFileName file={file} />}
+      <RowLevelErrorsContent
+        table={<ValidationErrorTable invalidations={errors} />}
+        title='Review your CSV file'
+        resetUpload={handleEnrollmentsReset}
+      />
+      </>
+    )
+  }
+
+  const renderSchemaInvalidations = (invalidations: SchemaInvalidation[]): JSX.Element => {
+    const errors = invalidations.map(
+      (invalidation, i) => <Typography key={i}>{invalidation.message}</Typography>
+    )
+    return (
+      <>
+      {file !== undefined && <CSVFileName file={file} />}
+      <ErrorAlert messages={errors} tryAgain={handleEnrollmentsReset} />
+      </>
+    )
+  }
+
+  const getUploadContent = (): JSX.Element => {
+    if (schemaInvalidations !== undefined) return renderSchemaInvalidations(schemaInvalidations)
+    if (rowInvalidations !== undefined) return renderRowValidationErrors(rowInvalidations)
+
+    const fileData =
+    `${REQUIRED_ENROLLMENT_HEADERS.join(',')}
+    student,studenta
```

Comment:
`studenta` is real test user we should be giving that in the sample csv file. Change it like userf

Reply:
I mean to say we shouldn't be giving that user since the user may accidentally adding to course, causing some noise.

Reply:
Sure, this has been in the feature since it was first started I think (before the expansion). How is this? https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/301/commits/ac2422b35af39bada44d0899ca7d43200974c1f2

Reply:
the change looks good. Change what you can here, not to worry of changing in other case, we can deal with that once we step into that code.